lib/rubocop/ast/node/if_node.rb

```diff
@@ -25,6 +25,13 @@ def unless?
         keyword == 'unless'
       end
 
+      # Checks whether the `if` node has an `then` clause.
+      #
+      # @return [Boolean] whether the node has an `then` clause
+      def then?
+        loc.begin&.is?('then')
```

Comment:
Could you please rebase and use the new `loc_is?` here? 🙏

Reply:
Sure! I've updated it!